index.js

```diff
@@ -64,3 +65,41 @@ module.exports = function (cb) {
 		});
 	}
 };
+
+module.exports.sync = function () {
+	if (cache) {
+		return cache;
+	}
+
+	var env = process.env;
+	var locale = env.LC_ALL || env.LANGUAGE || env.LANG || env.LC_MESSAGES;
+
+	if (locale) {
+		cache = getLocale(locale);
+		return cache;
+	}
+
+	var stdout = process.platform === 'win32' ?
+		execFileSync('wmic', ['os', 'get', 'locale'], {encoding: 'utf8'}) :
+		execFileSync('locale', {encoding: 'utf8'});
+
+	var getAppleLocale = function () {
+		cache = execFileSync('defaults', ['read', '-g', 'AppleLocale']).strim() || 'en_US';
```

Comment:
`strim()` => `trim()` and missing encoding option. Fixed in https://github.com/sindresorhus/os-locale/commit/cf130b6e79b0daf5b38092eda0fcca91a48340ed